doc/design/proposals/telemetry.md

```diff
@@ -0,0 +1,20 @@
+# Telemetry Support
+
+This is the design proposal for vSphere Integrated Containers Engine's initial integration with Vmware Analytics Cloud.  This data provides Vmware with simple usage metrics.  Currently, vSphere Integrated Containers Engine will piggyback on one of the data field that vSphere 6 and 6.5 are already collecting, specifically the OS full name used in the VM.  By providing a custom OS name for both the VCH and container VM, Vmware can determine the install base of VIC, the distribution of version installed, and the number of container VM used in a VCH.
+
+In future updates, VIC will be updated to collect data on which features are most commonly used to guide Vmware development plans.
+
+## Design Proposal
+
+vSphere collects a VM's OS full name by querying this data from the guest tools running inside the VM.  VIC installs a minimal toolbox inside the VCH's appliance VM and all container VMs created by VIC.  The toolbox will be updated to provide the following OS name for both VMs:
+
+| VM Type | OS Full Name |
+| --- | --- |
+| VCH | Photon OS - VCH v.v.v |
+| container VM | Photon OS - Pico v.v.v |
+
+In the above, v.v.v represents the actual version.  These strings can either added to the toolbox as part of the build process, or they can be added to the VM's guestinfo, which toolbox can read.  These strings will help differentiate the VCH from the container VM.  It will also provide Vmware with a distribution of VIC versions install base.
+
+For future feature-usage based data collection, VIC will need to provide another mechanism to collect data.  The initial proposal for this near future work is to divide this problem into a collection problem and a data upload problem.  The collection can be solved by updating the personality server and the portlayer servers to track feature usage metrics and saving it in the VM's guestinfo.  The data upload can be performed by vicAdmin.  The Vmware Analytics Cloud collector within vSphere will need to have it's data schema updated for these new data fields.
```

Comment:
It's likely that we'll be able to provide an alternative mechanism for persisting logs and metrics in the medium term - just FYI.